Documentation/DevelopmentTips.md

```diff
@@ -101,6 +101,21 @@ Alternatively, if you're working on an `mscorlib.dll` bug:
 	  adb push external/mono/mcs/class/lib/monodroid/mscorlib.dll \
 	    /data/data/Mono.Android_Tests/files/.__override__
 
+# Windows Build Notes
+
+Currently Windows avoids many of the macOS dependencies by downloading a zip bundle
+of binaries previously built on macOS. This speeds up the build and enables
+development on Windows, in general.
+
+A simple way to ensure you have the needed dependencies on Windows is to install
+Visual Studio 2017 (> 15.3.x) along with the Xamarin workload. This will ensure you have
+the correct version of Xamarin.Android, the Android SDK, and Java needed.
+
+It also is worth noting that opening `Xamarin.Android.sln` in Visual Studio tends
```

Comment:
@jonpryor I wanted to ask about this, if you have even seen it. Simply _opening_ the sln seems to lock `xa-prep-tasks` and `BootstrapTasks`.

I have seen similar issues when I contributed to Xamarin.Forms about 6 months ago, and their build is quite a bit simpler.

Reply:
>  Simply opening the sln seems to lock `xa-prep-tasks` and `BootstrapTasks`.

I have not seen it. Sounds like something for @kzu to possibly chime in on? :-D